monkestation/code/modules/liquids/code/liquid_systems/liquid_turf.dm

```diff
@@ -115,8 +115,15 @@
 			var/datum/reagent/R = r
 			liquids.reagent_list[R.type] = R.volume
 			liquids.total_reagents += R.volume
-			alpha_setting += R.opacity * R.volume
-			alpha_divisor += 1 * R.volume
+			alpha_setting += max(R.opacity * R.volume, 0.0001)
+			alpha_divisor += max(1 * R.volume, 0.0001)
+
+		//Ew but this is needed for edge cases in the new evaporation as its "technically" possible for numbers to balance out to 0 and cause division issues
+		if(alpha_divisor == 0)
+			alpha_divisor = 1
+		if(alpha_setting == 0)
+			alpha_setting = 1
+
 		liquids.alpha = min(round(alpha_setting / alpha_divisor), 255)
```

Comment:
Whatever the floor needs to be in there, change it as needed. 
But if you make sure that it's rounded to a non-zero number, it can't end up as 0
```suggestion
		liquids.alpha = clamp(round(alpha_setting / alpha_divisor), 0.1, 255)
```